File: Datasets/HumanEval/HumanEval_Java/11_all.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class StringXor {
    /**
     * Input are two strings a and b consisting only of 1s and 0s.
     * Perform binary XOR on these inputs and return result also as a string.
     * >>> string_xor('010', '110')
     * '100'
     * >>> string_xor('0001', '1100')
     * '1101'
     * @param a a string consisting only of 1s and 0s.
     * @param b a string consisting only of 1s and 0s.
     * @return the binary XOR of a and b
     */
    public static String stringXor(String a, String b) {

File: Cleared_Dataset/HumanEval_Py/11_all.py
from typing import List


def string_xor(a: str, b: str) -> str:
    """ Input are two strings a and b consisting only of 1s and 0s.
    Perform binary XOR on these inputs and return result also as a string.

    args:
        a: first string
        b: second string
    
    returns:
        string containing result of binary XOR on a and b

    examples:
    >>> string_xor('010', '110')
    '100'
    >>> string_xor('0010', '1110')
    '1100'
    """
